lib/node_modules/@stdlib/utils/configdir/lib/configdir.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+// MODULES //
+
+var path = require( 'path' );
```

Comment:
`var join = require( 'path' ).join;`

This change will affect subsequent usage below.